chain/actors/adt/adt.go

```diff
@@ -57,77 +47,3 @@ func (m *MapOpts) Equal(o *MapOpts) bool {
 
 	return true
 }
-
-func MapOptsForActorCode(c cid.Cid) (*MapOpts, error) {
```

Comment:
nixed in favor of methods on the generated actor accessors that return the correct values for BitWidth and HashFunction